packages/shared/src/components/utilities.tsx

```diff
@@ -249,3 +254,10 @@ export const getContextBottomPosition = (
 
   return { x: right, y: bottom + 4 };
 };
+
+export const fadedBackgroundStyle: CSSProperties = {
+  background: 'radial-gradient(ellipse, #c029f088 0%, #c029f000 400px)',
+  backgroundSize: '1200px 500px',
+  backgroundPosition: 'center -270px',
+  backgroundRepeat: 'no-repeat',
+};
```

Comment:
Nice one!
Noticed the bg-repeat is also a tailwind class perhaps we can use simply that one?

Reply:
Sharp eyes! 🔭  Let me simplify this quickly! 🚢

Reply:
I tried to refactor but it seems the class is getting overpowered.

![Screenshot 2023-01-15 at 10 58 17 PM](https://user-images.githubusercontent.com/13744167/212549691-15ceea74-7c63-4e66-9cca-6502fb0ffad7.png)